wp-content/themes/core/components/pagination/css/pagination.pcss

```diff
@@ -28,3 +28,44 @@
 
 	}
 }
+
+/*
+Pagination Single
+ */
+
+.pagination--single {
+	max-width: var(--grid-8-col);
+	margin: var(--spacer-60) auto;
+	border-top: 1px solid var(--color-border);
+	border-bottom: 1px solid var(--color-border);
+	padding: var(--spacer-40) 0;
+}
+
+.pagination__list {
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: space-between;
+	gap: var(--grid-gutter);
+
+	.pagination__item {
+		font-size: 14px;
+		font-weight: bold;
+
+		@media (--viewport-medium) {
+			font-size: var(--font-size-body-small);
+		}
+	}
+
+	.pagination__item-link--next {
+		font-family: var(--font-family-sans-serif);
+
+		&:after {
+			padding-left: 10px;
+			margin-top: 5px;
+		}
+	}
+
+	.pagination__item-link--previous {
+
+	}
```

Comment:
empty class

Reply:
In regards to these empty classes, they are declared in the markup for BEM purposes. Does our build system strip them? Is it helpful to leave them there for the next person? Or would you still remove it?

Reply:
We generally try to remove them. The build system won't automatically, no. If a project needs to add an additional selector (based on existing classes or otherwise), it's easy enough to do so.